chemprop/cli/utils/parsing.py

```diff
@@ -119,6 +119,38 @@ def make_datapoints(
     dataset.
     Parameters
     ----------
+    smiss : list[list[str]] | None
+        a list of lists of SMILES strings of shape ``n x j``, where ``j`` is the number of molecule
+        components per datapoint and ``n`` is the total number of datapoints
+    rxnss : list[list[str]] | None
+        a list of lists of reaction SMILES strings of shape ``n x k``, where ``k`` is the number of
+        reaction components per datapoint and ``n`` is the total number of datapoints
```

Comment:
the outer index should be defined first, i.e., `n` gets defined before `j` or `k` here